File: etapa2_fortran/prueba.f90
program prueba
    implicit none
    integer :: unidad
    print *, "FORTRAN OK"
    open(newunit=unidad, file="/mnt/c/polyflow/salida/prueba_fortran.txt", status="replace")
    write(unidad, *) "hola desde fortran"
    close(unidad)
end program prueba

File: etapa3_java/Prueba.java
import java.io.FileWriter;
import java.io.IOException;

public class Prueba {
    public static void main(String[] args) throws IOException {
        System.out.println("JAVA OK");
        FileWriter escritor = new FileWriter("/mnt/c/polyflow/salida/prueba_java.txt");
        escritor.write("hola desde java\n");
        escritor.close();
    }
}
